ibm_watson/natural_language_understanding_v1.py

```diff
@@ -1947,19 +3495,24 @@ class Features():
 
     def __init__(self,
                  *,
+                 classifications: 'ClassificationsOptions' = None,
                  concepts: 'ConceptsOptions' = None,
                  emotion: 'EmotionOptions' = None,
                  entities: 'EntitiesOptions' = None,
                  keywords: 'KeywordsOptions' = None,
-                 metadata: object = None,
+                 metadata: 'MetadataOptions' = None,
```

Comment:
are we allowed to change this from an object to `MetadataOptions`

Reply:
MetadataOptions itself is just an empty object. No code changes should result from this change although it is still an odd one. This is likely to implement new properties in the future